### PR TITLE
32 add pagination to the fandom shop

### PR DESCRIPTION
**Pull request by Hladikes**

### CODE REVIEW

File: routes/index.tsx [hunk 1]
```diff
@@ -1,38 +1,74 @@
...
+import IconCart from "@/components/IconCart.tsx";
+import { formatBalance, sanitizeUri } from "@/utils/data.ts";
+import { graphql } from "@/utils/indexer.ts";
+import { Item } from "@/utils/types.ts";
+import { useComputed } from "@preact/signals";
+import { aspectRatio } from "@twind/aspect-ratio";
+import { extendFields, getClient } from "@kodadot1/uniquery";
+import { tw } from "twind";
+
+const client = getClient("ahp");
```
Comment (vikiival) [inline routes/index.tsx:14]: Use chain const exposed via `Deno.env.get("CHAIN")`

File: routes/index.tsx [hunk 2]
```diff
@@ -1,38 +1,74 @@
...
+    const url = new URL(req.url);
+
+    const countQuery = client.itemCountByCollectionId(collectionId);
+    const count = await graphql<CountResult>(
+      countQuery.query,
+      countQuery.variables,
+    )
+      .then((result) => result?.itemCount?.totalCount);
+
+    const itemsPerPage = 12;
```
Comment (vikiival) [inline routes/index.tsx:44]: Also would put this to ENV as `PAGINATION`

**Review by vikiival — COMMENTED** [review]
very small

File: routes/index.tsx (hunk 1 above)
Comment (Hladikes) [reply routes/index.tsx]: done

File: routes/index.tsx (hunk 2 above)
Comment (Hladikes) [reply routes/index.tsx]: done

**Review by vikiival — APPROVED** [review]
Amazing as usual!

**Pull request merged**